adapters/Mintegral/MintegralAdapter/GADMediationAdapterMintegral.m

```diff
@@ -55,39 +111,51 @@ + (GADVersionNumber)adapterVersion {
+                         completionHandler:(nonnull GADRTBSignalCompletionHandler)completionHandler {
+    if (completionHandler) {
+        completionHandler([MTGBiddingSDK buyerUID],nil);
+    }
 }
 
-- (void)loadBannerForAdConfiguration:(GADMediationBannerAdConfiguration *)adConfiguration
-                   completionHandler:(GADMediationBannerLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding banner ad.
+- (void)loadRewardedAdForAdConfiguration:(GADMediationRewardedAdConfiguration *)adConfiguration completionHandler:(GADMediationRewardedLoadCompletionHandler)completionHandler {
+    _rewardedAd = [[GADMAdapterMintegralRewardedAdLoader alloc]init];
+    [_rewardedAd loadRewardedAdForAdConfiguration:adConfiguration
+                                completionHandler:completionHandler];
 }
 
-- (void)loadInterstitialForAdConfiguration:
-            (GADMediationInterstitialAdConfiguration *)adConfiguration
-                         completionHandler:
-                             (GADMediationInterstitialLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding interstitial ad.
+- (void)loadInterstitialForAdConfiguration:(GADMediationInterstitialAdConfiguration *)adConfiguration completionHandler:(GADMediationInterstitialLoadCompletionHandler)completionHandler {
+    _interstitialAd = [[GADMAdapterMintegralInterstitialAdLoader alloc]init];
+    [_interstitialAd loadInterstitialForAdConfiguration:adConfiguration
+                                      completionHandler:completionHandler];
 }
 
-- (void)loadNativeAdForAdConfiguration:(GADMediationNativeAdConfiguration *)adConfiguration
-                     completionHandler:(GADMediationNativeLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding native ad.
+- (void)loadBannerForAdConfiguration:(GADMediationBannerAdConfiguration *)adConfiguration completionHandler:(GADMediationBannerLoadCompletionHandler)completionHandler {
+    _bannerAd = [[GADMAdapterMintegralBannerAdLoader alloc]init];
+    [_bannerAd loadBannerForAdConfiguration:adConfiguration
+                        completionHandler:completionHandler];
 }
 
-- (void)loadRewardedAdForAdConfiguration:(GADMediationRewardedAdConfiguration *)adConfiguration
-                       completionHandler:
-                           (GADMediationRewardedLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding rewarded ad.
+- (void)loadNativeAdForAdConfiguration:(GADMediationNativeAdConfiguration *)adConfiguration completionHandler:(GADMediationNativeLoadCompletionHandler)completionHandler {
+    _nativeAd = [[GADMAdapterMintegralNativeAdLoader alloc]init];
+    [_nativeAd loadNativeAdForAdConfiguration:adConfiguration completionHandler:completionHandler];
+}
+
++ (void)setAdmobChannel {
+    //This is to call the mintegral ad sdk private method, which is only used to mark the official aggregation channel from admob
```

Comment:
Can update the comment to:
```
// Set up the AdMob aggregation channel for server statistic collection purposes.
```

Please verify if it is accurate.

Reply:
OK, I've made changes and committed.